packages/server/graphql/queries/helpers/suggestedIntegrationHelpers.ts

```diff
@@ -79,7 +79,7 @@ export const getTeamIntegrationsByUserId = async (
   return (
     res
       // jira integrations are making it through that don't have a projectKey
-      .filter((res) => res.service !== 'jira' || res.projectKey)
+      .filter((res) => permLookup[res.service] && (res.service !== 'jira' || res.projectKey))
```

Comment:
This fixes the following issue:

- Remove this change
- Add the Jira & GitHub integrations
- Integrate a task with Jira and then remove the Jira integration
- Open the dropdown menu on a task card
- See that it shows the recently used Jira project, even though we're no longer integrated with Jira

I felt that it's best to filter here in the helper file rather than in `suggestedIntegration` as everything in `suggestedIntegration` uses `teamIntegrationsByTeamId`, which is what is returned from this function. By filtering at the source, we don't need to filter elsewhere.